src/meta/model/migration/src/m20241121_101830_table_engine.rs

```diff
@@ -0,0 +1,38 @@
+use sea_orm_migration::prelude::{Table as MigrationTable, *};
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .alter_table(
+                MigrationTable::alter()
+                    .table(Table::Table)
+                    .add_column(ColumnDef::new(Table::Engine).string().not_null())
```

Comment:
To avoid backward compatible issue we need to specify a default value for this column. Or we can make it nullable instead.